content/pt-br/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Obtendo uma Imagem de um Registro Privado
+content_type: task
+weight: 100
+---
+
+<!-- overview -->
+
+Esta página mostra como criar uma Pod que usa uma 
+{{< glossary_tooltip text="Secret" term_id="secret" >}} para puxar uma imagem
+de um registro de imagens de contêiner privado ou de um repositório. Existem muitos registros privados em uso. 
+Esta tarefa usa o [Docker Hub](https://www.docker.com/products/docker-hub)
+como exemplo de registro.
+
+{{% thirdparty-content single="true" %}}
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}}
+
+* Para fazer este exercício, você precisa da ferramenta de linha de comando `docker`, e um
+  [Docker ID](https://docs.docker.com/docker-id/) para o qual você conheça a senha.
+* Se você estiver usando um registro de contêiner privado diferente, você precisa da 
+ferramenta de linha de comando para esse registro e suas informações de login. 
+
+<!-- steps -->
+
+## Logar em Docker Hub
+
+Em seu laptop, você deve autenticar-se em um registro para puxar uma imagem particular.
+
+Use a ferramenta `docker` para logar-se no Docker Hub. Veja a seção _log in_ do
+[Contas do Docker ID](https://docs.docker.com/docker-id/#log-in) para mais informações.
+
+```shell
+docker login
+```
+
+Quando solicitado, insira o seu Docker ID, e então a credencial que você deseja usar (token de acesso,
+ou a senha para o seu Docker ID).
+
+O processo de login cria ou atualiza um arquivo `config.json` que contém um token de autorização. Analise [como o Kubernetes interpreta esse arquivo](/docs/concepts/containers/images#config-json). 
+
+Veja o arquivo `config.json`:
+
+```shell
+cat ~/.docker/config.json
+```
+
+A saída contém uma seção semelhante a esta:
+
+```json
+{
+    "auths": {
+        "https://index.docker.io/v1/": {
+            "auth": "c3R...zE2"
+        }
+    }
+}
+```
+
+{{< note >}}
+Se você usar o armazenador de credenciais do Docker, você não vai ver essa entrada `auth`, mas sim uma `credsStore` com o nome do armazenador como valor.
+{{< /note >}}
+
+## Crie um segredo com base nas credenciais existentes {#registry-secret-existing-credentials}
+
+Um cluster de Kubernetes usa o Segredo do tipo `kubernetes.io/dockerconfigjson` para autenticar com um registro de contêineres
+para pegar uma imagem privada.
+
+Se você já executou `docker login`, você pode copiar
+essa credencial para o Kubernetes:
+
+```shell
+kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Se você precisar de mais controle (por exemplo, para definir um namespace ou um novo
+segredo) então você pode personalizar o segredo antes de armazená-lo.
```

Comment:
The sentence in brackets is not accurately translated. The original one mentions "to set a namespace or a label in the new secret" and parts of it were lost. Could you please review it?

Also, Secret should not be translated here, as it still refers to the Kubernetes object.